File: calculate_sum.py
def calculate_sum(arr):
    total = o            # NameError at runtime: name 'o' is not defined
    for num in arr:
        total += num
    return total

numbers = [1, 2, 3, 4, 5]
result = calculate_sum (numbers)
print("Sumin Python :", result)  # Typo acceptable, but spacing is fine

File: calculateSum.js
function calculateSum(arr) {
    let total = o;        // ReferenceError at runtime: o is not defined
    for (let num of arr) { total += num; }
    return total;
}

let numbers = [1, 2, 3, 4, 5];
let result = calculate Sum (numbers); // SyntaxError: Unexpected identifier 'Sum'
console.log("Sum in JavaScript:", result);
